docs/docs/workflows/terraform-module.md

```diff
@@ -0,0 +1,156 @@
+# Use the Terraform module
+You can manage a Constellation cluster through Terraform.
+The module package is available as part of the [GitHub release](https://github.com/edgelesssys/constellation/releases/). It consists of a convenience module for each cloud service provider (`{csp}-constellation`) that combines the IAM (`infrastructure/{csp}/iam`), infrastructure (`infrastructure/{csp}`), and constellation (`constellation-cluster`) modules.
+
+## Prerequisites
+- a Linux / Mac operating system
+- a Terraform installation of version `v1.4.4` or above
+
+## Quick setup
+The convenience module allows setting up a Constellation cluster with a single module. It's easiest to consume the module through a remote source, as shown below.
+This allows to upgrade the cluster to a newer Constellation version by simply updating the module source.
+
+1. Create a directory (workspace) for your Constellation cluster.
+  ```bash
+  mkdir constellation-workspace
+  cd constellation-workspace
+  ```
+
+1. Create a `main.tf` file to call the CSP specific Constellation module.
+
+  <tabs groupId="csp">
+
+  <tabItem value="azure" label="Azure">
+
+  ```
+  locals {
+    module_version = "v0.0.0" // insert Constellation version here.
+  }
+
+  module "azure-constellation" {
+    source = "https://github.com/edgelesssys/constellation/releases/download/${local.module_version}/terraform-module.zip//modules/azure-constellation"
```

Comment:
```suggestion
    source = "https://github.com/edgelesssys/constellation/releases/download/${local.module_version}/terraform-module.zip//modules/azure-constellation"
```

Testing this, I noticed the following: 
```
╷
│ Error: Variables not allowed
│ 
│   on main.tf line 5, in module "aws-constellation":
│    5:     source = "https://github.com/edgelesssys/constellation/releases/download/${local.module_version}/terraform-module.zip//modules/aws-constellation"
│ 
│ Variables may not be used here.
╵
```

We could instead use a bash variable and output the contents to a file.

Reply:
Also, I think our path is not `modules`, but `terraform-module`. I still need to verify that once the actual release is out though.

Reply:
Good point. I actually had some issues referencing `terraform-module` and it ended up working with `modules` so we might want to zip the files in a `modules` directory.

Reply:
since it's a singe variable I'd rather go back to `<version>` // please insert your version here
instead of overcomplicating.

Reply:
this URL works:
`https://github.com/edgelesssys/constellation/releases/download/v2.13.0/terraform-module.zip//terraform-module/aws-constellation`

Reply:
so the current path is correct. thanks for testing!